src/reducers/melindaCustomMergeFunctions.js

```diff
@@ -0,0 +1,27 @@
+// Custom functions for Melinda merge
```

Comment:
File could be named as something more common, like 'utils.js' and there you could collect all functions what can be used in any reducer later on. (checkDropSubfields and/or compareSubValues)